catalog/imaging/imaging-plugin-nitf/src/test/java/org/codice/alliance/plugin/nitf/PreStoragePluginTest.java

```diff
@@ -81,7 +84,23 @@ public void setUp()
         when(contentItem.getId()).thenReturn("101ABC");
         when(contentItem.getInputStream()).thenAnswer(invocationOnMock -> getInputStream(GEO_NITF));
         when(contentItem.getMimeTypeRawData()).thenReturn(NitfPreStoragePlugin.NITF_MIME_TYPE.toString());
-        when(contentItem.getSize()).thenReturn(5L*1024L*1024L);
+        when(contentItem.getSize()).thenReturn(5L * 1024L * 1024L);
+    }
+
+    @Test
+    public void testRunTimeException() throws IOException, PluginExecutionException {
+        NitfRenderer nitfRenderer = mock(NitfRenderer.class);
+
+        NitfPreStoragePlugin nitfPreStoragePlugin1 = new NitfPreStoragePlugin() {
+            @Override
+            protected NitfRenderer getNitfRenderer() {
+                return nitfRenderer;
+            }
+        };
+
+        when(nitfRenderer.render(any(ImageSegment.class))).thenThrow(IllegalArgumentException.class);
```

Comment:
❗️ We should throw a `RuntimeException` here to make sure all sub-classes will be caught.